src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -2208,13 +2208,7 @@ void M(object o1, object? o2)
 }";
 
             var comp = CreateCompilation(source, options: WithNonNullTypesTrue());
-            comp.VerifyDiagnostics(
-                // (10,25): warning CS8606: Possible null reference assignment to iteration variable
```

Comment:
Uh, where did these warnings go?

Reply:
They've become W warning, which is suppressed in code above (`#pragma warning disable CS8600`)

---
In reply to: [355704457](https://github.com/dotnet/roslyn/pull/40207#discussion_r355704457) [](ancestors = 355704457)

Reply:
Ah, I understand.

---
In reply to: [355709463](https://github.com/dotnet/roslyn/pull/40207#discussion_r355709463) [](ancestors = 355709463,355704457)